src/components/InputsForms/Forms.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import "./Forms.scss"
+
+interface FormsProps{
+    sentence: string,
+
+}
+
+
+function Forms({sentence}: FormsProps) {
+  return (
+    <form className='create'>
+        <label>Nome</label>
+        <input className='create--text'
+            type='text'
+            required
+        />
+        <label>Email</label>
+        <input className='create--text'
```

Comment:
-- is variation not element.
you could use, create__input--text and create__input--text-area